project_national/fact_sheets_category_5.yml

```diff
@@ -0,0 +1,277 @@
+        costs: &costs_mshp_ductless_SEER_15_9-5_HSPF
+        # Source: Custom regression by Brennan Less. Ductless heat pump project costs were regressed on nameplate tons,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ductless MSHP n=173, HSPF median 11, range 9.3–14.2. Implicitly includes multiple zones and electrical upgrades.
+        # Regression results in costs that are 15–30% higher than costs from The Heat Pump Store as documented in https://redwoodenergy.net/wp-content/uploads/2021/02/Pocket-Guide-to-All-Electric-Retrofits-of-Single-Family-Homes.pdf 
+        # LBNL data were not sufficient to include a relationship between SEER or HSPF and cost, so cost deltas for higher 
+        # and lower HSPF were added using the relationship between HSPF and cost for ductless MSHPs in REMDB (https://remdb.nrel.gov/).
+          - value: 2472.60
+            multiplier: Fixed (1)
+          - value: 320.42
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: *lifetime_ductless_MSHP_elec_backup 
+
+      - *HP_cooling_efficiency
+      - *100_percent_conditioned
+
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+  - upgrade_name: Higher efficiency ductless MSHP w/ elec backup  #also provides SEER 15 cooling measure profile
+    # Measure 6
+    options:
+      # MSHP
+      - &ductless_MSHP_HigherEfficiency_elec_backup
+        option: HVAC Heating Efficiency|MSHP, SEER 20, 11 HSPF, CCHP # Changed to Max Load
+        apply_logic: *ductless_MSHP_elec_backup
+        costs: &costs_mshp_ductless_SEER_20_11_HSPF
+        # Source: Custom regression by Brennan Less. Ductless heat pump project costs were regressed on nameplate tons,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ductless MSHP n=173, HSPF median 11, range 9.3–14.2. Implicitly includes multiple zones and electrical upgrades.
+        # Regression results in costs that are 15–30% higher than costs from The Heat Pump Store as documented in https://redwoodenergy.net/wp-content/uploads/2021/02/Pocket-Guide-to-All-Electric-Retrofits-of-Single-Family-Homes.pdf 
+        # LBNL data were not sufficient to include a relationship between SEER or HSPF and cost, so cost deltas for higher 
+        # and lower HSPF were added using the relationship between HSPF and cost for ductless MSHPs in REMDB (https://remdb.nrel.gov/).
+          - value: 2472.60
+            multiplier: Fixed (1)
+          - value: 320.42
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: *lifetime_ductless_MSHP_elec_backup 
+
+      - *HP_cooling_efficiency
```

Comment:
see comments above

Reply:
@lixiliu using package_apply_logic should fix all comments above if I understand it correctly